subprojects/test-fixtures/src/main/kotlin/org/gradle/kotlin/dsl/fixtures/ProjectBuilder.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl.fixtures
+
+import org.gradle.api.Project
+
+import org.gradle.testfixtures.ProjectBuilder
+
+import java.io.File
+
+
+fun File.newProjectBuilder(name: String = this.name): ProjectBuilder =
```

Comment:
:o: a `File` extension seems wrong here. I'd define this as a regular factory function like `newProjectBuilderWith(projectDir: File, name: String = projectDir.name)`